### PR TITLE
Bump omniscidb current development version to 5.8.0; Add omniscidb 5.6 to CI

### PR DESCRIPTION
**Pull request by guilhermeleobas**
As title.

### CODE REVIEW

**Comment by guilhermeleobas** [comment]
On hold until omniscidb dev image gets updated to v5.8

File: rbc/tests/test_omnisci_column_arguments.py [hunk 1]
```diff
@@ -83,7 +83,7 @@ def text_rbc_copy_cccc_rowmul(x, x2, x3, x4, m, y, y2, y3, y4):
                           'cursor(i4,i8,f8,f4)'])
 def test_copy(omnisci, use_default, inputs):
     if use_default:
-        omnisci.require_version((5, 6), 'Requires omnisci-internal PR 5403')
+        omnisci.require_version((5, 7), 'Requires omnisci-internal PR 5403')
```
Comment (pearu) [inline rbc/tests/test_omnisci_column_arguments.py:86]: Why this change? PR 5403 has been merged a long time ago.

File: rbc/tests/test_omnisci_column_arguments.py [hunk 2]
```diff
@@ -117,7 +117,7 @@ def test_copy(omnisci, use_default, inputs):
 
 @pytest.mark.parametrize('kind', ['1', '11', '111', '211', '221', '212', '13', '32', '34', '242'])
 def test_ct_binding_constant_sizer(omnisci, kind):
-    omnisci.require_version((5, 5, 5), 'Requires omniscidb-internal PR 5274')
+    omnisci.require_version((5, 7), 'Requires omniscidb-internal PR 5274')
```
Comment (pearu) [inline rbc/tests/test_omnisci_column_arguments.py:120]: Why this change?

File: rbc/tests/test_omnisci_column_default.py [hunk 3]
```diff
@@ -17,7 +17,7 @@ def rbc_udtf_default_sizer_1(c, sizer, o):
 
 
 def test_python_fn_with_default_sizer(omnisci):
-    omnisci.require_version((5, 6), "Requires omniscidb-internal PR 5403")
+    omnisci.require_version((5, 7), "Requires omniscidb-internal PR 5403")
```
Comment (pearu) [inline rbc/tests/test_omnisci_column_default.py:20]: Why this change?

File: rbc/tests/test_omnisci_column_default.py [hunk 4]
```diff
@@ -66,7 +66,7 @@ def test_python_fn_with_default_sizer(omnisci):
 
 @pytest.mark.parametrize("suffix,kind,expected", typs)
 def test_default_sizer(omnisci, suffix, kind, expected):
-    omnisci.require_version((5, 6), "Requires omniscidb-internal PR 5403")
+    omnisci.require_version((5, 7), "Requires omniscidb-internal PR 5403")
```
Comment (pearu) [inline rbc/tests/test_omnisci_column_default.py:69]: Why this change?

**Review by pearu — COMMENTED** [review]
It is not clear to me why omniscidb version test requirements are changed. @guilhermeleobas , can you elaborate?

**Comment by guilhermeleobas** [comment]
> It is not clear to me why omniscidb version test requirements are changed. @guilhermeleobas , can you elaborate?

When I introduce the tests for the default sizer, I changed a few tests in the omniscidb repository. Since omniscidb 5.6 was released prior to this change, newer versions of rbc assume the tests in the "new" format. Also, we never caught this issue because we never ran omniscidb 5.6 on CI. 

I guess this is a perfect example of why one must never modify tests in the server 😅

**Review by pearu — APPROVED** [review]
> .. I guess this is a perfect example of why one must never modify tests in the server sweat_smile

Makes sense.

LGTM!
